processing/src/main/java/org/apache/carbondata/processing/loading/BadRecordsLoggerProvider.java

```diff
@@ -33,6 +33,16 @@
    * @return
    */
   public static BadRecordsLogger createBadRecordLogger(CarbonDataLoadConfiguration configuration) {
+    return createBadRecordLogger(configuration, false);
+  }
+  /**
```

Comment:
please format this comment. empty line should be added at line:38 and removed in line:43